src/arns.lua

```diff
@@ -91,6 +123,12 @@ function arns.addRecord(name, record)
 	end
 end
 
+--- Gets paginated records
+--- @param cursor string|nil The cursor to paginate from
+--- @param limit number The limit of records to return
+--- @param sortBy string The field to sort by
+--- @param sortOrder string The order to sort by
+--- @return PaginatedTable<Record> The paginated records
```

Comment:
```suggestion
--- @return PaginatedTable<Record> paginatedRecords The paginated records
```

Reply:
addressed a good chunk in [cebb3cb](https://github.com/ar-io/ar-io-network-process/pull/175/commits/cebb3cba577aa5a936939eb7bc99e01da74ebb9f)